test/enigma_test.rb

```diff
@@ -129,4 +141,29 @@ def test_it_knows_what_is_valid_input
 		assert_equal false, enigma.is_valid_input?(valid_message_6, valid_key_6, invalid_date_6)
 	end
 
+	def test_it_can_encrypt_a_message_with_no_additional_input
+		enigma_1 = Enigma.new
+		enigma_1.stubs(:get_date_of_today).returns("040895")
+		enigma_1.stubs(:make_random_keys).returns("02715")
+
+		expected = {:encryption=>"keder ohulw", :key=>"02715", :date=>"040895"}
+		assert_equal expected, enigma_1.encrypt("hello world")
+
+		enigma_2 = Enigma.new
+		enigma_2.stubs(:get_date_of_today).returns("070620")
+		enigma_2.stubs(:make_random_keys).returns("02715")
+
+		expected = {:encryption=>"nib udmcxpu", :key=>"02715", :date=>"070620"}
+		assert_equal expected, enigma_2.encrypt("hello world")
+	end
+
+	def test_it_can_make_random_keys
+		enigma = Enigma.new
+    random_keys = enigma.make_random_keys
+    assert_instance_of String, random_keys
+    assert_equal 5, random_keys.length
+    assert_includes 0...100_000, random_keys.to_i
+		assert_equal true, random_keys.each_char.all? { |key| ("0".."9").include?(key) }
+	end
+
```

Comment:
It looks like there's still something going on with invisible tabs?
Maybe a linter would help?